sources/osgShader/Compiler.js

```diff
@@ -936,15 +935,16 @@ define( [
                 'uniform mat4 ProjectionMatrix;',
                 'uniform mat4 NormalMatrix;',
                 '',
-                'varying vec4 VertexColor;',
-                'varying vec3 FragNormal;',
-                'varying vec3 FragEyeVector;',
+                'varying vec4 VertexColor;',// investigate lowp
```

Comment:
ha, yes. in fact, following this morning conversation, it's a bigger job that just enabling precision here (need check input, need change arraybuffer, etc)
 maybe an osgjs Issue ? 
(better handling of precsion in vertex attribute)
and then removing code comments here